crash_alert/crash_alert.py

```diff
@@ -0,0 +1,24 @@
+from message_senders.slack_message_sender import SlackMessageSender
+from message_senders.sns_message_sender import SnsMessageSender
+import time
+
+class CrashAlert():
+    def __init__(self, t, slack_webhook):
```

Comment:
```suggestion
    def __init__(self, min_time_interval, slack_webhook):
```
Feel free to change it to something else, but would be nice to have something more descriptive than `t`